functests/performance/hugepages.go

```diff
@@ -77,4 +84,100 @@ var _ = Describe("[performance]Hugepages", func() {
 	// 		Expect(string(nrHugepages)).To(Equal("128"))
 	// 	})
 	// })
+
+	Context("Basic hugepages usage verification", func() {
+		var testpod *corev1.Pod
+
+		AfterEach(func() {
+			err := testclient.Client.Delete(context.TODO(), testpod)
+			Expect(err).ToNot(HaveOccurred())
+
+			err = pods.WaitForDeletion(testclient.Client, testpod, 60*time.Second)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("Pod should be able to run process with using hugepages", func() {
```

Comment:
can you adapt this test to take values from the performance profile, instead of using hardcoded paths for hugepages?

Reply:
I thought about using profile for it, but in this test I'm creating the pod with 1G hugepages only, so i don't need to have mutable links..
but if you think we can have cluster with 2M hugepages only - so yeah, need to upgrade test..

Reply:
Do you want to do it under the current PR or under the separate PR?

Reply:
I've updated code - now it works for 1G and 2M hugepages